### PR TITLE
fix(client): runtime error with icons/spacer component

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/components/helpers/toggle-button.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export default function ToggleButton({
 }: ButtonProps): JSX.Element {
   const checkIconStyle = {
     height: '15px',
+    paddingTop: '5',
```
Comment (ShaunSHamilton) [inline client/src/components/helpers/toggle-button.tsx:35]: Is the Spacer not used in many more places than just this Component?

Comment (moT01) [reply client/src/components/helpers/toggle-button.tsx:35]: I don't believe so, that's [this other spacer](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/src/components/helpers/Spacer.js). I only found this one spot anyway - maybe I'm mistaken, I only did a quick search.

Comment (ShaunSHamilton) [reply client/src/components/helpers/toggle-button.tsx:35]: Quite right. It triggers my OCD a bit we have such similar components with the same name, though.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Pull request merged**